client.go

```diff
@@ -821,6 +821,8 @@ func (sc *SnowthClient) DoRequestContext(ctx context.Context, node *SnowthNode,
 
 	var hdr http.Header
 
+	var status int
```

Comment:
HTTP status codes are a text value

Reply:
They are, but Go's standard library returns an int representation of the status code with when the http.Client runs the request. The string version is returned too, but this int is what we are checking to see if it is between 400 and 500.